.github/workflows/ci.yml

```diff
@@ -123,7 +123,7 @@ jobs:
     needs: gen_llhttp
     strategy:
       matrix:
-        pyver: [3.7, 3.8, 3.9, '3.10']
+        pyver: ['3.7', '3.8', '3.9', '3.10']
```

Comment:
This is also unnecessary, plus isn't even related to the action versions.

Reply:
yeah, sorry. I intended to do this one in a separate commit. I just changed it for consistency with '3.10', 3.10 without quotes tries to install python 3.1 (because it thinks it's a string).